<?xml version="1.0"?>
<launch>
  <node pkg="mbzirc2020_task1_tasks" type="ransac_line_fitting_node" name="ransac_line_fitting_node" output="screen" >
    <param name="target_point_sub_topic_name" value="/treasure/point_detected"/>
    <param name="cand_points2d_max_size" value="30"/>
    <param name="cand_points3d_max_size" value="30"/>
    <param name="ransac_3d_mode" value="False"/>
    <param name="target_point_maximum_disappear_time" value="0.8"/>
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mbzirc2020_task1_tasks)/config/task1_ransac_vis.rviz" />

</launch>
